apps/api/src/app/subscribers-v2/usecases/list-subscribers/list-subscribers.command.ts

```diff
@@ -0,0 +1,29 @@
+import { DirectionEnum } from '@novu/shared';
+import { IsOptional, IsString, IsEnum } from 'class-validator';
+import { CursorBasedPaginatedCommand } from '@novu/application-generic';
+
+export class ListSubscribersCommand extends CursorBasedPaginatedCommand {
+  @IsEnum(DirectionEnum)
+  @IsOptional()
+  orderDirection: DirectionEnum = DirectionEnum.DESC;
+
+  @IsEnum(['updatedAt', 'createdAt'])
+  @IsOptional()
+  orderBy: 'updatedAt' | 'createdAt' = 'createdAt';
```

Comment:
by the way, if those optional params have default we can remove the static default values from 
`apps/api/src/app/subscribers-v2/subscriber.controller.ts`

```suggestion
  @IsEnum(DirectionEnum)
  @IsOptional()
  orderDirection?: DirectionEnum = DirectionEnum.DESC;

  @IsEnum(['updatedAt', 'createdAt'])
  @IsOptional()
  orderBy?: 'updatedAt' | 'createdAt' = 'createdAt';
```